.yamllint.yaml

```diff
@@ -1,8 +1,39 @@
 ---
 
 rules:
+
+  brackets:
+    forbid: false
+    min-spaces-inside: 0
+    max-spaces-inside: 1
+
+  commas:
+    min-spaces-after: 1
+
   document-start: disable
+
+  float-values:
+    require-numeral-before-decimal: true
+
+  hyphens:
+    max-spaces-after: 1
+
+  indentation:
+    indent-sequences: whatever
+    spaces: consistent
+
+  key-duplicates:
+    forbid-duplicated-merge-keys: true
+
   line-length:
+    allow-non-breakable-words: true
+    allow-non-breakable-inline-mappings: true
     max: 120
     level: warning
+
+  new-lines:
+    type: unix
+
+  trailing-spaces: {}
+
```

Comment:
These are rules around the spacing/placement of YAML fields. We already implicitly follow these rules, but this makes it abundantly clear that these are our conventions.

The most important ones here are:
- `allow-non-breakable-words`: Allow lines to surpass 120 characters if they can't be easily broken up (i.e. URLs).
- `new-lines.type`: When people develop on Windows, the encoding for new lines (`CRLF`) is different from Unix/Linux standards (`LF`). This can have impacts on formatting and sha256 sums, so we don't allow them.
- `forbid-duplicated-merge-keys`: All information pertaining to an entry should be found under one unique entry. Having multiple identical keys in a dictionary is apparently allowed in YAML representations. I really don't want that.
- `brackets.min|max-spaces-inside`: This is my personal preference, but I like seeing this: `[_'some-entry', 'another-entry'_]`, but this: `['some-entry', 'another-entry']` is also acceptable.
- The `indentation` rules specify that we don't care whether you use 2 or 3 or 4 spaces to denote entries under a key, so long as you use them consistently throughout the file, therefore the following would not be OK:
```YAML
entry1:
  - something1
  - something2
entry2:
    - otherthing1
    - otherthing2
```

Reply:
Thanks for this explanation, interesting! Although, also, sorry you had to write all this, I just meant to say I'm not 100% familiar but it looks legit, so take my approval with a grain of salt on this particular file. It looks almost the same a xscen so I assume it will work fine.